basic/tests/test_prover.rs

```diff
@@ -185,25 +185,29 @@ fn prove_fibonacci() {
     type Challenge = Val; // TODO
     type PackedChallenge = Challenge; // TODO
 
-    type MyMds = MdsMatrixBabyBear;
-    let mds = MyMds {};
+    type Mds16 = CosetMds<Val, 16>;
+    type Mds32 = CosetMds<Val, 32>;
```

Comment:
So it's now using a 32-element permutation for linear hashes and 16-element for Merkle tree compression. I don't know if it's optimal, more of an experiment. After we optimize hashes (mainly vectorization) and try newer ones like Monolith, we can say with more certainty what's optimal.